thoth/license_solver/solver.py

```diff
@@ -36,22 +36,25 @@
 
 def detect_license(
     input_data: Union[Dict[str, Any], str, List[str], List[Dict[str, Any]]], raise_on_error: bool = True
-) -> None:
+) -> Dict[str, Any]:
     """Run license-solver from thoth-solver."""
     try:
         solver = Solver()
 
         if type(input_data) == dict or type(input_data) == str:
```

Comment:
It might be better to use isinstance here:

```suggestion
        if isinstance(input_data, dict) or isinstance(input_data, str):
```

Reply:
uuuuu, in the next PR, I will change it :smile: